src/atoms/tags/tag.styles.ts

```diff
@@ -0,0 +1,35 @@
+import { Styled } from '../../theme';
+
+export const CustomTag = Styled.div<{ type: string, isOn:boolean }>`
```

Comment:
@goshatravin let us know what you think about the below so we can find best way of doing this 👍 

I wonder if this approach is the best because assuming you have array of some strings and you want to map over to show tags i would need some special logic in the frontend to decide which tag to show on which index. 

The way this component was made originally in frontend repo was something like this:

```

export const Tag: FC<TagProps> = ({
  height, width, fontSize, fontWeight, index, selected, tag, setTag,
}) => {
  const { night } = useAppState();

  return (
    <div
      role="button"
      onClick={() => {
        setTag(tag);
      }}
      tabIndex={index}
      style={{
        width, height, fontSize, fontWeight,
      }}
      className={cn(`tag tag--${index % 4}`, { night, [`tag--${index % 4}--selected`]: selected })}
    >
      <div className="text" style={{ fontSize }}>
        {tag.title}
      </div>
    </div>
  );
};


```

The index would be used to determine which variation to show, and then if lets say there were 10 tags to display it would simply loop the colors again starting from first variation. With that i think you could just have an array with 6 object entries with background and border properties. Then use the passed index as index for that array `arrayOfBackgroundAndBorderVariations[i % 6]`

you would avoid some repetition with that especially in the `tag.tsx`

Reply:
Yes this makes sense, there is a fair bit of repetition, I just followed the figma names/options for now. But yeah let me know how you guys wanna handle this and I will get it changed.

Reply:
@shan8851  first of all , i like new structure, lets start using it for all new components , and ye @ArinCantCode  100% cool idea , @shan8851  how much time do we need to change it ?

Reply:
@goshatravin ok great, just makes everything a little cleaner and nice especially when not working on multiple screens/large display.

Regarding the change, I don't think too long, I will get to this later this evening or first thing in the morning